AnswerKingAPI.yml

```diff
@@ -82,6 +94,33 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProblemDetails'
+    post:
+      tags:
+        - Tags
+      summary: Unretire a retired Tag.
+      parameters:
+        - name: id
+          in: path
+          description: ''
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '201':
```

Comment:
This should also return the unretired tag, shouldn't it?

Reply:
changed